### PR TITLE
Add a --latest option to the snapshots:load method to load the latest available snapshot

### PR DESCRIPTION
**Pull request by ssmusoke**
I took this approach instead of the initial suggestion in https://github.com/spatie/laravel-db-snapshots/issues/94 as the intent of the option is much clearer

### CODE REVIEW

File: src/Commands/Load.php [hunk 1]
```diff
@@ -30,7 +30,13 @@ public function handle()
             return;
         }
 
-        $name = $this->argument('name') ?: $this->askForSnapshotName();
+        $use_latest_snapshot = $this->option('latest') ?: false;
```
Comment (freekmurze) [inline src/Commands/Load.php:33]: Use camelCase for variables.

File: src/Commands/Load.php [hunk 2]
```diff
@@ -30,7 +30,13 @@ public function handle()
...
+
+        if ($use_latest_snapshot) {
+            $name = $snapShots->last()->name;
+        } else {
```
Comment (freekmurze) [inline src/Commands/Load.php:37]: Avoid using `else`, use ternaries instead.

File: src/Commands/Load.php [hunk 3]
```diff
@@ -30,7 +30,13 @@ public function handle()
             return;
         }
 
-        $name = $this->argument('name') ?: $this->askForSnapshotName();
+        $use_latest_snapshot = $this->option('latest') ?: false;
+
+        if ($use_latest_snapshot) {
+            $name = $snapShots->last()->name;
```
Comment (freekmurze) [inline src/Commands/Load.php:36]: What happens when there are no snapshots? Add a test for that case as well.

**Comment by freekmurze** [comment]
Tests for L5.8 seem to be failing. Could you take a look at that? I'm also ok with just dropping support for L5.8.

File: src/Commands/Load.php (hunk 3 above)
Comment (ssmusoke) [reply src/Commands/Load.php:36]: That is already caught by the first check in the command

File: src/Commands/Load.php (hunk 2 above)
Comment (ssmusoke) [reply src/Commands/Load.php:37]: I was conflicted here since the else has a ternary operator already

**Comment by ssmusoke** [comment]
> Tests for L5.8 seem to be failing. Could you take a look at that? I'm also ok with just dropping support for L5.8.

The failures are due to a bug in the Symfony Console below 4.3.4 fixed here https://github.com/symfony/symfony/pull/32806 that is pulled in by the prefer-lowest option which causes 14 tests to fail

File: src/Commands/Load.php (hunk 1 above)
Comment (ssmusoke) [reply src/Commands/Load.php]: Done

**Comment by ssmusoke** [comment]
@freekmurze just a ping for additional feedback

File: src/Commands/Load.php (hunk 3 above)
Comment (freekmurze) [reply src/Commands/Load.php:36]: Right 🙂

**Comment by freekmurze** [comment]
I'll polish this on master, thanks!

**Pull request merged**